File: uva/java/uva_11332/src/uva_11332/Uva_11332.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package uva_11332;

import java.util.Scanner;

/**
 *
 * @author DELL
 */
public class Uva_11332 {

    /**
     * @param args the command line arguments
     */
    static Scanner scan = new Scanner(System.in);
    public static void main(String[] args) {
        // TODO code application logic here
        
     long input,ans;

    while(scan.hasNext())
    {
        input = scan.nextLong();
        if(input == 0) break;

         ans = input % 9;

         if(ans == 0) ans = 9;

         System.out.println(ans);
    }
    }
    
}

File: uva/c_cpp/uva-11332.c
#include <stdio.h>

int main()
{
    long long input,ans;

    while(scanf("%lld",&input) == 1)
    {
        if(input == 0) break;

         ans = input % 9;

         if(ans == 0) ans = 9;

         printf("%lld\n",ans);
    }
    return 0;
}
